src/libraries/System.Transactions.Local/tests/OleTxTests.cs

```diff
@@ -469,6 +469,8 @@ private static void Test(Action action)
             return;
         }
 
+        TransactionManager.ImplicitDistributedTransactions = true;
```

Comment:
This is changing a process-wide setting.  Will that lead to race conditions based on how tests are executed?  For things where we're changing process-wide state, we generally use RemoteExecutor to isolate the changes.

Reply:
Yeah, this is why the test that the switch works (exception unless enabled) is in the class fixture, so that it executes before everything else.

Good point though, I'll move this out using RemoteExecutor.